Library/Formula/dnscrypt-proxy.rb

```diff
@@ -121,7 +121,7 @@ def plist; <<-EOS.undent
         <array>
           <string>#{opt_sbin}/dnscrypt-proxy</string>
           <string>--ephemeral-keys</string>
-          <string>--resolvers-list=#{share}/dnscrypt-proxy/dnscrypt-resolvers.csv</string>
+          <string>--resolvers-list=#{pkgshare}/dnscrypt-resolvers.csv</string>
```

Comment:
Should be `opt_pkgshare` here really. Same principle as the `opt_sbin`; If we leave it as `share` the path is versioned and people need to replace the `plist` every update which people definitely do not do :smile: